test-tests/shared/src/test/scala/zio/test/TestAspectSpec.scala

```diff
@@ -196,9 +197,17 @@ object TestAspectSpec extends ZIOBaseSpec {
     } @@ timeout(1.nanos)
       @@ failure(diesWithSubtypeOf[TestTimeoutException]),
     testM("timeout reports problem with interruption") {
-      assertM(ZIO.never.uninterruptible *> ZIO.unit, equalTo(()))
-    } @@ timeout(10.millis, 1.nano)
-      @@ failure(diesWith(equalTo(interruptionTimeoutFailure)))
+      for {
+        testClock <- ZIO.environment[TestClock]
+        liveClock <- Live.make(testClock)
+        spec = testM("uninterruptible test") {
+          for {
+            _ <- (testClock.clock.adjust(11.milliseconds) *> ZIO.never).uninterruptible
+          } yield assertCompletes
+        } @@ timeout(10.milliseconds, 1.nanosecond) @@ failure(diesWith(equalTo(interruptionTimeoutFailure)))
+        result <- isSuccess(spec.provideManaged(Managed.succeed(liveClock)))
```

Comment:
`provideManaged(Managed.succeed(...))` maybe we need `provide`?

Reply:
Yes, I was thinking the same thing. I will do a PR and add all the other variants.